Source/WebKit/NetworkProcess/storage/CacheStorageCache.cpp

```diff
@@ -375,8 +398,17 @@ void CacheStorageCache::removeAllRecords()
 
 void CacheStorageCache::close()
 {
+    assertIsOnCorrectQueue();
+
     m_records.clear();
     m_isInitialized = false;
 }
+    
+void CacheStorageCache::assertIsOnCorrectQueue()
```

Comment:
Could you mark this as inline? Just want to increase the likelihood we avoid function calls in release builds.